Common/include/option_structure.hpp

```diff
@@ -951,20 +951,60 @@ inline TURB_FAMILY TurbModelFamily(TURB_MODEL model) {
     case TURB_MODEL::SA_COMP:
     case TURB_MODEL::SA_E_COMP:
       return TURB_FAMILY::SA;
-    case TURB_MODEL::SST:
-    case TURB_MODEL::SST_SUST:
+    case TURB_MODEL::SST: case TURB_MODEL::SST_SUST:
       return TURB_FAMILY::KW;
   }
   return TURB_FAMILY::NONE;
 }
 
+/*!
+ * \brief SST Options
+ */
+enum class SST_OPTIONS {
+  NONE,        /*!< \brief No SST Turb model. */
+  V1994,       /*!< \brief 1994 Menter k-w SST model. */
+  V2003,       /*!< \brief 2003 Menter k-w SST model. */
+  V1994m,       /*!< \brief 1994m Menter k-w SST model. */
+  V2003m,       /*!< \brief 2003m Menter k-w SST model. */
+  MODIFIED,    /*!< \brief Modified k-w SST model. */
+  SUST,        /*!< \brief Menter k-w SST model with sustaining terms. */
+  VORTICITY,   /*!< \brief Menter k-w SST model with vorticity production terms. */
+  KL,          /*!< \brief Menter k-w SST model with Kato-Launder production terms. */
+  UNCERTAINTY, /*!< \brief Menter k-w SST model with uncertainty quantification modifications. */
+};
+static const MapType<std::string, SST_OPTIONS> SST_Options_Map = {
+  MakePair("NONE", SST_OPTIONS::NONE)
+  MakePair("V1994m", SST_OPTIONS::V1994m)
+  MakePair("V2003m", SST_OPTIONS::V2003m)
+  MakePair("V1994", SST_OPTIONS::V1994)
+  MakePair("V2003", SST_OPTIONS::V2003)
+  //MakePair("MODIFIED", SST_OPTIONS::MODIFIED)
```

Comment:
Just to be clear about this: the currently implemented 1994 model still has a 'divergence source term', which should be present only when the model is NOT modified. 
So, for now I can add this term to the V1994 model, and then we remove it when we switch to the literature V1994 model.

Reply:
I also think that we should not stick with the name "V1994m" and V2003m". I think we should have only 1994 and 2003 as base versions, and "Modified" as an additional option. For now I am making "V1994m" as much as possible an alias of "V1994" + "modified"

Reply:
I know, but we are dropping terms with `k` which I guess makes it closer to the modified.

V?m would only be the ways for the user to specify, internally we would always convert to V? + m.
The problem with the user specifying V? + m is how to default m. Because we want to default to modified but then a user would also need to explicitly say `UNMODIFIED`.
So this can either be specified as (V1994 V1994m V2003 V2003m) or as (V1994 V2003 MODIFIED UNMODIFIED) or as (V1994 V2003 UNMODIFIED).
In the last one the absence of UNMODIFIED implies m, and we get the desired defaulting behavior.